components/Projects.jsx

```diff
@@ -1,35 +1,37 @@
+import Project from './Project'
+
 export default () => (
-  <div>
-    <h2>Our Projects</h2>
-    <h3>News 451</h3>
-    <p>
-      News 451 is a minimalistic web application that allows users to see the
-      recent news without clutter or ads. Users can gauge reliability and bias
-      of their news using our custom machine learning model.
-    </p>
-    <img
-      src="https://cdn.discordapp.com/attachments/583367531406950431/585948245461696531/GroupNews451Combined.png"
-      alt="News 451 website"
+  <div style={{ paddingTop: '15px' }}>
+    <Project
+      name="News451"
+      img="https://cdn.discordapp.com/attachments/583367531406950431/591331216989356063/news451big.png"
+      description="News 451 is a minimalistic web application that allows users to see the recent news without clutter or ads. Users can gauge reliability and bias of their news using our custom machine learning model."
+      icon="https://pbs.twimg.com/profile_images/839515548333420545/ThVZeSAJ_400x400.jpg"
+      prizeDescription="1st place at the Sixth Baltimore Hackathon"
+      prize="$1000"
+    />
+    <hr
+      style={{
+        marginLeft: '-50vw',
+        marginRight: '-50vw',
+      }}
+    />
```

Comment:
This actually creates 2 horizontal lines:
![image](https://user-images.githubusercontent.com/24739403/59874539-6f072100-936c-11e9-8315-473f81afe5fa.png)

Reply:
Wow! I did not notice that on my 1080p monitor it looks like one line.